ExternalLookup/.run/ExternalLookup.run.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+    <configuration name="ExternalLookupJob" type="Application" factoryName="Application" nameIsGenerated="true">
+        <envs>
+            <env name="SECURITY_PROTOCOL" value="SASL_PLAINTEXT" />
+            <env name="TOKEN_URL" value="http://localhost:1852/realms/local-development/protocol/openid-connect/token" />
+            <env name="KAFKA_CLIENT_ID" value="default-access" />
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.ExternalLookupJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id ExternalLookup-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic-main-source ExternalLookupInputTopic --output-kafka-topic EnrichStreamOutputTopic" />
```

Comment:
Possibly missing postfix for output-kafka-topic.

Reply:
Also fixed on my branch